addons/mail/static/src/components/chatter_container/chatter_container.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.ChatterContainer" owl="1">
         <div class="o_ChatterContainer flex-grow-1 d-flex" t-att-class="{ 'o-isInFormSheetBg mx-auto': props.isInFormSheetBg }" t-attf-class="{{ props.className }}" data-command-category="mail" t-ref="root">
-            <t t-if="chatter">
+            <t t-if="chatter and chatter.thread">
```

Comment:
Ok as fix (minimal code change while fixing the issue)

Not to change in this PR: spinner should be responsibility of `Chatter`, not `ChatterContainer`. There might be technical reasons to have `ChatterContainer` but no `Chatter`, but we should clean code so that `ChatterContainer` is just a technical layer for using messaging components below that have proper UI concerns like this "Please wait..."